TemporalioSamples.sln

```diff
@@ -35,6 +35,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TemporalioSamples.Schedules
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TemporalioSamples.ActivityHeartbeatingCancellation", "src\ActivityHeartbeatingCancellation\TemporalioSamples.ActivityHeartbeatingCancellation.csproj", "{CD0D6B7E-2076-4771-AF5B-E1EFB82A44B3}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TemporalioSamples.Polling.Frequent", "src\Polling\Frequent\TemporalioSamples.Polling.Frequent.csproj", "{6935B8AC-160F-463D-BE03-AD6FF31585A3}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Polling", "Polling", "{AE21E7F4-B114-4761-81B1-8FA63E9F6BB8}"
```

Comment:
I don't think this project is accurate in the `dotnet sln list`

Reply:
I made a solution folder called `Polling` to group those three samples together. 

![solution-folder](https://github.com/temporalio/samples-dotnet/assets/287641/b135f31b-8cd5-4937-b81b-b885224f8cfd)

Reply:
I see. Seems it happened with other multi-project sample folders we had too I guess.

Reply:
Yeah that guid `2150E333-8FDC-42A3-9474-1A3956D46DE8` is a solution folder.
https://github.com/VISTALL/visual-studio-project-type-guids